frontend/src/component/layout/Error/Error.tsx

```diff
@@ -55,6 +55,7 @@ export const Error: VFC<IErrorProps> = ({ error }) => {
                     <ConditionallyRender
                         condition={showZendeskButton}
                         show={<ZendeskButton />}
+                        elseShow={undefined}
```

Comment:
I think it's more idiomatic to return null than undefined in react. I checked the code and the ConditionallyRender swaps undefined to null under the hood but in the spirit of "don't make me think" I'd go for null here.

Reply:
Same for the other places where we do this pattern

Reply:
Yes, but in this place prop `customButton` expects something `|| undefined`

Reply:
Ah so it's for the next step of the migration when we do codemods. Those things are worth mentioning in the PR description since it's not obvious.